build.sbt

```diff
@@ -73,6 +74,19 @@ lazy val clientPlay = module("be-kind-rewind-play", Some("client-play"))
   )
   .dependsOn(core)
 
+lazy val clientAkkaHttp = module("be-kind-rewind-akka-http", Some("client-akka-http"))
+  .settings(
+    fork := true,
+    baseDirectory in run := file("."),
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka" %% "akka-http"        % Version.akkaHttp,
+      "com.typesafe.akka" %% "akka-stream"      % Version.akka  % Provided,
+      "com.typesafe.akka" %% "akka-actor-typed" % Version.akka  % Provided,

```

Comment:
Note) `Provided` is to support two different major versions of akka: 

* v2.5.x: Classic (Non-typed) Actor
* v2.6.x: Typed Actor